plaidml/BUILD

```diff
@@ -36,6 +36,7 @@ plaidml_cc_library(
         "//pmlc:all_passes_and_dialects",
         "//pmlc/target/demo",
         "//pmlc/target/intel_gen",
+        "//pmlc/target/intel_gen_ocl_spirv",
```

Comment:
This shouldn't block this PR, but I think we need a better organization scheme for our targets.  :-)